src/mixs/schema/mixs.yaml

```diff
@@ -3703,13 +3703,14 @@ slots:
     title: Hazard Analysis Critical Control Points (HACCP) guide food safety term
     examples:
     - value: tetrodotoxic poisoning [FOODON:03530249]
+    - value: tetrodotoxic poisoning[FOODON:03530249]; neurotoxic shellfish poisoning[FOODON:03530246]
     keywords:
     - food
     - term
     slot_uri: MIXS:0001215
     multivalued: true
     range: string
-    pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
+    pattern: ^.+\s*\[FOODON:\d+\](;\s*\[FOODON:\d+\])*$
```

Comment:
@turbomam @only1chunts

Reply:
Thanks! Please use the pattern  `^(\S[^\r\n]*) [FOODON:\d{7,8}]$` instead of `^.+\s*\[FOODON:\d+\](;\s*\[FOODON:\d+\])*$` or see my notes on dynamic enumerations.

Reply:
Did you intentionally remove the white-space between the label and the term id? I don't _think_ that's consistent with other ontology term patterns in MIxS

Reply:
@turbomam 
For the white space, do you mean if it should be "lead poisoning [FOODON_03530243]" vs "lead poisoning[FOODON_03530243]:" 

So, the white space is supposed to be there? I thought I had it set to be valid with or without it... does it matter? If so, I'll make sure I correct it. Just tell me which is correct.

Looking at the submission schema the white space should be there. So I can make that update to the regex. 

Question, for my own ability to learn
`^(\S[^\r\n]*) [FOODON:\d{7,8}]$.`  vs
`^.+\s*\[FOODON:\d+\](;\s*\[FOODON:\d+\])*$`

Is your multivalued?

Reply:
The regex validator I use says that "lead poisoning [FOODON_03530243]" is not valid with `^.+\s*\[FOODON:\d+\](;\s*\[FOODON:\d+\])*$` ... are you sure that's right??  Or am I missing something about the formatting of the value for "lead poisoning [FOODON_03530243]" ?

<img width="1437" alt="Screenshot 2024-07-09 at 4 42 01 PM" src="https://github.com/GenomicsStandardsConsortium/mixs/assets/44581429/d15dde22-9e2f-4b21-8559-3f7da17d3d77">

Reply:
From your comment here : https://github.com/GenomicsStandardsConsortium/mixs/pull/802#issuecomment-2218318449

> ^(\S[^\r\n]*) [FOODON:\d{7,8}]$


> I f we want to use pattern-only validation, I suggest we go with that.

That regex `^(\S[^\r\n]*) [FOODON:\d{7,8}]$`
is showing me that  "lead poisoning [FOODON:03530243]" is invalid too... :(

Reply:
<img width="1440" alt="Screenshot 2024-07-09 at 4 53 27 PM" src="https://github.com/GenomicsStandardsConsortium/mixs/assets/44581429/f11ebf4f-8293-43e1-9b7b-b63b6242f12b">
<img width="1444" alt="Screenshot 2024-07-09 at 4 53 40 PM" src="https://github.com/GenomicsStandardsConsortium/mixs/assets/44581429/884f9712-adb3-47ee-85c3-087a8cc484a4">

Reply:
decision, this should work ^(\S[^\r\n]*) [FOODON:\d{7,8}]$

I'll test this and confirm then finish this PR.

Discussed 12/03

Reply:
I forgot to escape the square brackets around FOODON with backslashes \[F... etc